src/watertap_contrib/reflo/unit_models/crystallizer_effect.py

```diff
@@ -159,9 +159,9 @@ def build(self):
         )
 
         self.overall_heat_transfer_coefficient = Var(
-            initialize=0.1,
+            initialize=1e3,
             bounds=(0, None),
-            units=pyunits.kilowatt / pyunits.m**2 / pyunits.degK,
+            units=pyunits.watt / pyunits.m**2 / pyunits.degK,
```

Comment:
Good catch-- I remember the only thing that was really changed previously that would have impacted this issue was the crystallization property model, where units were changed from kW to W. Thus, I am assuming that this was one of the main issues (which I figured could have been caught by an assert_units_consistent).